base/src/com/google/idea/blaze/base/sync/projectview/LanguageSupport.java

```diff
@@ -128,6 +129,7 @@ public static boolean validateLanguageSettings(
       supportedLanguages.addAll(syncPlugin.getSupportedLanguagesInWorkspace(type));
     }
     supportedLanguages.add(LanguageClass.GENERIC);
+    supportedLanguages.add(LanguageClass.RUST);
```

Comment:
This is pretty poor - it enables Rust support for EVERYONE, all the time. But I couldn't get "optional" Rust language support to work using a Rust equivalent of `AlwaysOnGoSyncPlugin`.